tests/data/lmod

```diff
@@ -1,29 +1,79 @@
 data = [
 {
-    "raw" : "<13>1 2016-02-18T14:52:49.399290+01:00 node2407 lmod: - lmod:: user=vsc40307, cluster=golett, arch=haswell, module=PSI/4.0b6-20160201-intel-2016a-mt-Python-2.7.11, fn=/apps/gent/CO7/haswell-ib/modules/all/PSI/4.0b6-20160201-intel-2016a-mt-Python-2.7.11, jobid=369145.master19.golett.gent.vsc",
+    "raw": "<13>1 2016-06-07T17:50:27.866534+02:00 node2159 lmod: - lmod:: username=vsc40307, cluster=delcatty, arch=sandybridge, jobid=434.master16.delcatty.gent.vsc, cmd=load, args=-GCC",
```

Comment:
include an entry for a command with multiple arguments, e.g. `swap`?

since we ran into a problem with spaces disappearing?